frontend/public/components/cluster-settings/global-config.tsx

```diff
@@ -31,6 +32,10 @@ class GlobalConfigPage_ extends React.Component<GlobalConfigPageProps, GlobalCon
     loading: true,
   }
 
+  getKnativeConfig(): plugins.GlobalConfig[] {
+    return plugins.registry.getGlobalConfig();
+  }
+
   componentDidMount() {
     let errorMessage = '';
     Promise.all(this.props.configResources.map((model: K8sKind) => {
```

Comment:
You should combine `configResources` with `plugins.registry.getGlobalConfig()`. If the extension should contain all the information necessary too make this handling generic.

Reply:
I'm not sure how to add configResources to the extension if that's what you're asking me to do. Can you point me to an example?

Reply:
Do not add `configResources` to the extension (although future we may want to push such info into the registry to have a single source of truth).

My suggestion was to do something like: `configResources.concat(plugins.registry.getGlobalConfig()).map(...`

This assumes the extension contains the same data structure as `configResources` so that everything is handled in the same way.

Reply:
`configResources` has a different data structure than the extension (it has the whole object and the model). That's why there's some data processing going on with `responses`; we don't need the whole object to get the links in.

Reply:
Got it! so ideally then you can concat the extensions with the post-processed responses for all the extensions whose flags are enabled. That is if the extension is made to contain the expected data structure required by this component.